public/static/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -0,0 +1,114 @@
+
+   ![](/static/img/gdrive-create-credentials.png)
+
+6. The newly generated **client ID** and **client secret** should be shown to
+   you now, and you can always come back to **Credentials** to fetch them.
+
+⚠️ It's safe to share **client ID** and **client secret** among your team. These
+credentials are used to
+[authorize](https://developers.google.com/drive/api/v2/about-auth) DVC to use
+your Google Drive.
+
+> Note that Google Drive API has usage limits/quotas per credentials in a
+> _project_ (which you can review in the
+> [OAuth consent screen](https://console.developers.google.com/apis/credentials/consent)).
+> Please keep this in mind when sharing them, or you may
+> [exceed the limits](https://developers.google.com/drive/api/v2/handle-errors?hl=ro#resolve_a_403_error_usage_limit_exceeded).
+
+## URL format for Google Drive remotes
+
+A GDrive DVC remote URLs is needed for the `dvc remote add` command. It can be
+constructed with a _base_, and an **optional** _path_ to a folder i.e.
+`gdrive://<base>/path/to/folder`. The base can be one of:
+
+1. `root` - indicates your topmost Google Drive directory.
+
+  ```dvc
+  $ dvc remote add -d mygdroot gdrive://root
+  $ dvc remote add mygdfolder gdrive://root/path/to/folder
+  ```
+
+2. Google Drive _Folder ID_
+
+  To obtain the unique _Folder ID_ in question, navigate into that folder in
+  your web browser, and find it in the address bar. For example, for
+  `https://drive.google.com/drive/folders/0AIac4JZqHhKmUk9PDA`:
+
+  ```dvc
+  $ dvc remote add mygdfolder gdrive://0AIac4JZqHhKmUk9PDA
+  $ dvc remote add mygdsubfolder \
+                   gdrive://0AIac4JZqHhKmUk9PDA/sub/path
+  ```
+
+  > Note that
+  > [shared drives](https://developers.google.com/drive/api/v2/about-shareddrives)
+  > can be referenced by _Folder ID_ in the same way.
+
+3. (rare) `appDataFolder` - special
+   [application-specific data](https://developers.google.com/drive/api/v2/appdata)
+   folder only accessible by your application and hidden from the user.
+
+  ```dvc
+  $ dvc remote add mygdappata gdrive://appDataFolder
+  ```
+
+## Configure the GDrive remote
+
+Use the `dvc remote modify` command to set the credentials for each `gdrive://`
+remote, for example:
+
+```dvc
+$ dvc remote add -d mygdfolder gdrive://root/path/to/folder
+$ dvc remote modify mygdfolder gdrive_client_id <client ID>
+$ dvc remote modify mygdfolder gdrive_client_secret <client secret>
+```
+
+## Authorization
+
+On the first usage of a GDrive [DVC remote](/doc/command-reference/remote),
+you'll be prompted to visit a web page to generate an access token, which will
+require you to sign into the corresponding Google account. The sign-in process
+will guide you through granting the necessary access permissions. On success,
+the token data will be cached in a Git-ignored directory located in
+`.dvc/tmp/gdrive-user-credentials.json`.
+
+⚠️ In order to prevent unauthenticated access to your Google Drive, do not share
```

Comment:
I would prefer to use here `unauthorized` instead of `unauthenticated` since user already authenticated by Google, but resources might be accessed without user's permission(authorization). 
https://medium.com/datadriveninvestor/authentication-vs-authorization-716fea914d55

Reply:
Yeah, "unauthorized" also works here (as a general term) but the problem in sharing the token AFAIK is that someone else would be "posing as you" so it's a matter of the user's identity (authentication). No strong opinion though, both terms are fine. Maybe "unauthorized" even sounds nicer (more common term).

Reply:
I agree with @MaxRis here. It is technically authenticated in this case. Usually this term is used on that level at least I believe.

Reply:
Changing to "unauthorized"